governance/pyth_staking_sdk/src/pyth-staking-client.ts

```diff
@@ -68,7 +68,9 @@ export class PythStakingClient {
   constructor(config: PythStakingClientConfig) {
     this.connection = config.connection;
     this.wallet = config.wallet;
-    this.provider = new AnchorProvider(this.connection, this.wallet, {
+
+    // {} as AnchorWallet is a workaround for AnchorProvider requiring a wallet
+    this.provider = new AnchorProvider(this.connection, this.wallet ?? {} as AnchorWallet, {
```

Comment:
does {} === undefined?

Reply:
no, but the check is for `this.wallet` not the `{}` in the provider.